core/lib/src/config/custom_values.rs

```diff
@@ -42,6 +43,47 @@ impl fmt::Display for SecretKey {
     }
 }
 
+#[derive(Copy, Clone)]
```

Comment:
Also derive `Debug`. We should also implement `PartialEq` with `self.value() == other.value()` as the implementation.